common/src/main/java/org/schemarepo/config/Config.java

```diff
@@ -90,6 +105,9 @@
 
     DEFAULTS.setProperty(DEFAULT_SUBJECT_VALIDATORS,"");
 
+    // Validation defaults. The read strategy class is "Always Fail" unless configured.
+    DEFAULTS.setProperty(VALIDATION_READ_STRATEGY_CLASS, "org.schemarepo.validation.AlwaysFailValidationStrategy");
```

Comment:
Is it true that this setting gets overridden if one activates the ```AvroValidationModule``` like this:

```
# register the avro vallidation module
schema-repo.module.avro-validation=org.schemarepo.config.AvroValidationModule
```

In which case, if one does configure the ```AvroValidationModule```, then there is no need to also set this:

```
# base "Can Read" validation strategy needed by the CanBeReadValidationStrategy and MutualReadValidationStrategy implementations
schema-repo.validation.can-read.strategy.class=org.schemarepo.validation.AvroCanReadValidationStrategy
```

Right?